buyer-api/src/services/notaryService.js

```diff
@@ -66,20 +43,24 @@ export const transferNotarizacionResult = async (notarizationResult) => {
    * the operation will add new job to the transaction queue to send the transaction to the network.
    */
   // data payload
-
-  // buyer's account id in the BatchPayments contracts
-  const { id: buyerId } = await getAccount();
-  const { dataExchange: dx } = config.contracts.addresses;
-
-  // search for data order
-  const dataOrder = await dataOrders.fetch(notarizationResult.orderId);
+  const {
+    price,
+    result: {
+      notarizationFee: fee,
+      orderId,
+      sellers,
+      lock,
+    },
+  } = await notarizations.fetch(notarizationRequestId);
 
   const payload = {
-    fromId: buyerId,
-    amount: dataOrder.price,
-    payData: getPayData(notarizationResult.sellers.map(s => s.id)),
-    lock: notarizationResult.lock,
-    metadata: `${dx}${notarizationResult.orderId}`,
+    amount: fromWib(price),
+    payData: numberToHex(getPayData(sellers.map(s => s.id))),
+    lock,
+    metadata: numberToHex(orderId),
```

Comment:
docs says that 
> metadata: hash of the dataEchange address concatenated with the orderId involved in the transaction
`numberToHex(orderId)` is not the same thing; is that ok? shall we update the docs?

Reply:
Yes, thanks for the heads up @genuinefafa. I have to change this but I wanted to make sure that we are on the same page. The metadata should be the concatenation of: 
* `dataExchangeAddress`
* `orderId`
* `notaryAddress`
Since this are the fields that are not persisted in the Payment information and we need to know why this payment is being done. Remember that the payData field sent in the transaction identifies the players that will receive the payment and it is not obfuscated in the data payload, so there is no need to include them in the metadata.
Also, the final string should be hashed to match the type expected by the contract.

Reply:
[last change](https://github.com/wibsonorg/buyer-sdk/pull/137/files#diff-603ce7769dc7cf1c8eb8454a042f3064) suit current docs, :tada:

Reply:
@manelli @Meketh FYI